tests/test_bulk_operations.py

```
@@ -0,0 +1,342 @@
+"""
+Test suite for bulk operations functionality.
+"""
+
+import pytest
+from unittest.mock import AsyncMock, patch
```

Ruff (0.11.9)

`unittest.mock.AsyncMock` imported but unused

Remove unused import: `unittest.mock.AsyncMock`

(F401)